src/openrct2-ui/windows/Ride.cpp

```diff
@@ -3180,49 +3184,60 @@ static void WindowRideOperatingMousedown(rct_window* w, rct_widgetindex widgetIn
     uint8_t upper_bound, lower_bound;
     switch (widgetIndex)
     {
+        case WIDX_MODE_TWEAK:
+            WindowRideOperatingTweakWindow(w, *ride);
```

Comment:
I don't quite see why you have used the suffix Window. That would imply that its creating a special operating tweak window but this is just opening the standard text input window.